test/junit/scala/collection/immutable/VectorTest.scala

```diff
@@ -181,4 +181,41 @@ class VectorTest {
     // test that type info is not lost
     val x: Vector[Char] = Vector[Char]().tapEach(f)
   }
+
+  @Test
```

Comment:
I think that, using property tests of Scalacheck, these tests can be written as:  
 ```Scala
prop { (v: Vector[Int], i: Int, j: Int) => 
    v.take(i).toArray.take(j) === v.take(i).iterator.take(j).toArray
}
```
That may be a wider set of tests than these ones.

Reply:
The existing tests were JUnit so that's what I used. I was hoping for scalacheck tests, too.